fastly/block_fastly_service_v1_acl.go

```diff
@@ -0,0 +1,129 @@
+package fastly
+
+import (
+	"fmt"
+	gofastly "github.com/fastly/go-fastly/fastly"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"log"
```

Comment:
Please separate std libraries from external packages.
```suggestion
	"fmt"
	"log"
	
	gofastly "github.com/fastly/go-fastly/fastly"
	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
```